Readme.md

```diff
@@ -58,7 +58,9 @@ jobs:
       apikey: ${{ secrets.DEFAULT_NUGET_PUBLISH_API_KEY }}
 ```
 
-The _publish.yml_ workflow is triggered on a tag pushed to any branch with the prefix `v` and should contain a [valid version number](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-basics), e.g. `v1.0.1`, which will be extracted and used to version the NuGet packages produced.
+The _publish.yml_ workflow is triggered on a tag pushed to any branch with the prefix `v` and should contain a [valid version number](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-basics), e.g. `v1.0.1`, which will be extracted and used to version the Orchard Core extension manifests and the NuGet packages produced.
+
+There is no configuration required for automated Orchard Core extension manifest versioning, all of the `Manifest.cs` files are looked up, and the existing `Version` properties are updated automatically inside the `Module` or `Theme` definition with the version pushed. Note that, the `Version` property should be present and leave the version number on the default value (0.0.1), this is because we don't actually need to keep manifest version changes in the code.
```

Comment:
You don't need comma after "Note that", because  it's not meaningful in itself. Also "this is because" is the start of a new sentence.  You could remove the "This is" instead, but I think this is less taxing.
```suggestion
There is no configuration required for automated Orchard Core extension manifest versioning, all of the `Manifest.cs` files are looked up, and the existing `Version` properties are updated automatically inside the `Module` or `Theme` definition with the version pushed. Note that the `Version` property should be present and leave the version number on the default value (0.0.1). This is because we don't actually need to keep manifest version changes in the code.
```

Reply:
Thank you for your suggestions.